src/System.Runtime.Serialization.Json/src/System.Runtime.Serialization.Json.csproj

```diff
@@ -15,9 +15,6 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <ProjectReference Include="$(SourceDir)System.Private.DataContractSerialization\src\System.Private.DataContractSerialization.csproj" />
```

Comment:
I believe this one might still be needed. Let's do it post step as this PR grows too big already